src/Appwrite/Platform/Workers/Webhooks.php

```diff
@@ -113,7 +121,21 @@ private function execute(array $events, string $payload, Document $webhook, Docu
         }
 
         if (false === \curl_exec($ch)) {
-            $this->errors[] = \curl_error($ch) . ' in events ' . implode(', ', $events) . ' for webhook ' . $webhook->getAttribute('name');
+            $dbForConsole->increaseDocumentAttribute('webhooks', $webhook->getId(), 'attempts', 1);
+            $webhook = $dbForConsole->getDocument('webhooks', $webhook->getId());
+            $errorCount = $webhook->getAttribute('attempts');
+            $lastErrorLogs = \curl_error($ch) . ' in events ' . implode(', ', $events);
+            $webhook->setAttribute('logs', $lastErrorLogs);
+            // $statusCode = curl_getinfo($ch, CURLINFO_RESPONSE_CODE);
+
+            if ($errorCount >= self::FAILEDATTEMPTSCOUNT) {
+                $webhook->setAttribute('enabled', false);
```

Comment:
You should probably also reset the attempts count here.. right ?

Reply:
It's already done when webhook is re-enabled from `projects.php` line 1059. It'd be redundant to do it here.

Reply:
We do that in the API endpoint when `status` is reset.